pydatastructs/trees/tests/benchmarks/test_binary_trees.py

```diff
@@ -0,0 +1,32 @@
+import random, timeit, functools, os, pytest
+from pydatastructs.trees.binary_trees import (BinarySearchTree)
+from pydatastructs.utils.misc_util import Backend
+import random
+
+@pytest.mark.xfail
+def test_BST_insert(**kwargs):
```

Comment:
```suggestion
def test_BinarySearchTree_insert(**kwargs):
```

Why only `insert` is benchmarked? Why not other operations?